backend/settings.js

```diff
@@ -90,6 +88,18 @@ function getDerivedSetting(settingName) {
                 'settings.yaml'
             );
 
+        case 'KEY_FILE':
+            return path.join(
+                getSetting('STORAGE_PATH'),
+                'privkey.pem'
+            );
+
+        case 'CERT_FILE':
+            return path.join(
+                getSetting('STORAGE_PATH'),
+                'fullchain.pem'
```

Comment:
is this the conventional name? I feel like I recall something with a `.crt` extension, but I don't really know